### PR TITLE
Add new options

### PR DESCRIPTION
**Pull request by strangelittlemonkey**

### CODE REVIEW

**Comment by ghoneycutt** [comment]
Hi @strangelittlemonkey Thanks for contributing! In order to add new parameters, they need to be documented in the README and have spec tests. Could you please add those and we can get this merged.

File: manifests/init.pp [hunk 1]
```diff
@@ -55,6 +58,9 @@
       $default_aliases            = 'files'
       $default_publickey          = 'files'
       $default_netgroup           = 'files'
+      $default_protocols          = 'files'
```
Comment (ghoneycutt) [inline manifests/init.pp]: you can remove these three lines

**Comment by strangelittlemonkey** [comment]
Sure, if I don't get a chance to today I'll do it over the weekend.

File: manifests/init.pp (hunk 1 above)
Comment (strangelittlemonkey) [inline manifests/init.pp]: Done

File: spec/classes/init_spec.rb [hunk 2]
```diff
@@ -417,6 +417,51 @@
     }
   end
 
+  context 'with protocols set' do
```
Comment (ghoneycutt) [inline spec/classes/init_spec.rb]: this is great!

**Comment by ghoneycutt** [comment]
Released in v1.7.0

**Pull request merged**